### ISSUE TITLE
Add more classifier operators

### ISSUE THREAD

**Issue by rhiever** [rhiever/tpot#128]
There are many more classifiers that we can add from sklearn. I have listed them below. We may not want to add some of these classifiers, however, especially the ones that have many evolvable parameters that affect their performance.

Before you proceed to implement a classifier, let's discuss it in this issue.

# AdaBoostClassifier

http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.AdaBoostClassifier.html

Evolvable parameters: learning_rate

Fixed parameters: n_estimators = 500, random_state = 42

# BernoulliNB

http://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.BernoulliNB.html

Evolvable parameters: alpha, binarize, fit_prior

Fixed parameters: none

# ExtraTreesClassifier

http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html

Evolvable parameters: criterion, max_features

Fixed parameters: n_estimators = 500, random_state = 42

# GaussianNB

http://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.GaussianNB.html

Evolvable parameters: none

Fixed parameters: none

# MultinomialNB

http://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.MultinomialNB.html

Evolvable parameters: alpha, fit_prior

Fixed parameters: none

# LinearSVC

http://scikit-learn.org/stable/modules/generated/sklearn.svm.LinearSVC.html

Evolvable parameters: C, loss, fit_intercept

Fixed parameters: random_state = 42

# PassiveAggressiveClassifier

http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.PassiveAggressiveClassifier.html

Evolvable parameters: C, loss, fit_intercept

Fixed parameters: random_state = 42

# SGDClassifier

http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html

Evolvable parameters: loss, alpha, penalty, fit_intercept, l1_ratio, eta0, power_t, learning_rate

Fixed parameters: random_state = 42
